docs/server.md

```diff
@@ -52,6 +52,45 @@ Select Node 10.x for the environment. Tachyon requires the S3 bucket and region
 
 ![](lambda-env.png)
 
+If you have the need of keeping your S3 bucket private, you can add an Environment Variable, with the key `S3_AUTHENTICATED_REQUEST` and the value `true` (case insensitive). In this case, it will be necessary to create an IAM Role for the function to be able to fetch data from the S3 Bucket.
+
+![](lambda-env-authenticated-request.png)
+
+Here is an example of IAM Role that should be created for this functionality to work. This policy allows the role to access any S3 Bucket with Get and Put permissions, and allows to create logs and log groups as well as Put log events and describe log streams:
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "s3:GetObject",
+                "s3:PutObject"
```

Comment:
FYI I don't think Tachyon needs `putObject`?

Reply:
Yeah... I don't think it needs also, from what I saw in the documentation and code... But the current Role created by the CF template uses it, I think I copied when I created the one in our environment: https://github.com/humanmade/tachyon/blob/master/cloudformation-template.json#L131-L132

Reply:
Oh doh yeah we should probably fix that!

Reply:
I can do it here :) let me add this